variables.tf

```diff
@@ -811,3 +811,15 @@ variable "tcp_keepalive_probes" {
   type        = number
   default     = 2
 }
+
+variable "deny_global_write_access" {
```

Comment:
why do you need this var, if you don't have producer roles you can just deny write access by default? Meaning you'll probably always want at least one producer role.

Reply:
I wanted to make it explicit it to avoid any confusion in our team. In this way, when you go our deployment it is much easier to understand we are blocking writes + adding producer roles. If we just add producer_roles I would say people won't imagine we are also blocking writes.

Reply:
Ok but the producer role is ignore unless `deny_global_write_access` is true which is also a little weird.

Maybe I'm just not sure what the problem is you're trying to solve, can you explain what does not work now and needs to work better?

Reply:
ok I think if add an example in the variable.md it will be a bit more clear.